analysis/src/CompletionBackEnd.ml

```diff
@@ -1135,6 +1135,32 @@ and getCompletionsForContextPath ~debug ~full ~opens ~rawOpens ~pos ~env ~exact
                      | _ -> false
                    else true)
         in
+
+        (* TODO(config-complete-builtins) This config and config resolution should come from `package` instead. *)
```

Comment:
`package` here is under `full.package` I believe, and is a concept used throughout the analysis. It represents the information needed from `rescript.json`.